gradle/sonar.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'org.sonarqube'
+
+def analysisMode = 'publish'
+//TODO(kzaikin): use preview mode in pull requests
+//if (new Boolean(System.env.TRAVIS_PULL_REQUEST)) {
+//    analysisMode = 'preview'
+//}
+
+sonarqube {
+    properties {
+        property 'sonar.analysis.mode', analysisMode
+        property 'sonar.host.url', System.env.SONAR_HOST_URL
+        property 'sonar.login', '11cab05437bb5e07221bd50afc1e694ea8f64b49'
```

Comment:
Maybe possible at https://travis-ci.org/spotbugs/spotbugs/settings

But it's not so good habit, I will also try to use travis-ci secret value.